### ISSUE TITLE
Option to have browser (dashboard) as main view

### ISSUE THREAD

**Issue by BoneheadFraggle** [thanksmister/android-mqtt-alarm-panel#84]
I'm using the device as a "alarm pad enabled dashboard".
It would be great to be able start the app in dashboard view (so it resumes the dashboard view after a restart)

It would also be great if it would switch from dashboard view to alarm view when the alarm changes state (I think it already does) and then switches back to dashboard view after a timer or 30 seconds after last alarm state change or similar.

**Comment by BoneheadFraggle** [comment]
Yeah, I guess you're right, it would. It's just that I use one phone with Wallpanel as a dashboard for Home Assistant. And just 1,5 meters away I have another phone with Alarm Panel.
I would be nice to combine those two and save a device.

Maybe I the Wallpanel and Alarm Panel could merge to one app?

Anyway, thanks a lot. Both apps are working really good.

**Comment by thanksmister** [comment]
There is already a dashboard view in the alarm panel, you are asking for the reverse, taking a dedicated alarm panel and making it a dashboard web page view with a secondary alarm panel feature. I don't want to diminish the purpose of the alarm panel.   Why not just add a alarm panel to your dashboard instead.
